pyproject.toml

```diff
@@ -309,13 +309,17 @@ filterwarnings = [
 ]
 
 markers = ["slow: marks tests as slow (deselect with '-m \"not slow\"')"]
+# must not end with `--cov` since that will treat the first argument you pass
+# to pytest as a coverage source! see
+# https://github.com/catalyst-cooperative/pudl/issues/3609
 addopts = """\
     --verbose \
     --pdbcls=IPython.terminal.debugger:TerminalPdb \
+    --cov \
     --cov-append \
     --cov-config=pyproject.toml \
     --cov-report=xml \
-    --cov"""
+    """
```

Comment:
Does adding this unescaped newline do anything weird?  I tried running a test locally and it seemed to work as expected so I guess not?

Reply:
Doesn't seem to!